src/partials/header.html

```diff
@@ -0,0 +1,47 @@
+<header class="header">
+  <div class="container header-top">
+    <nav class="header-nav">
+      <a href="./index.html" class="header-logo"
+        ><img
+          src="../images/logo.png"
```

Comment:
Нужно заменить 2 точки на 1, как сделано на примере снизу:

```suggestion
          src="./images/logo.png"
```